NEWS.md

```diff
@@ -4,6 +4,40 @@ PgBouncer changelog
 PgBouncer 1.22.x
 ----------------
 
+**2024-06-28  -  PgBouncer 1.22.2  -  "Into the new beginnings"**
```

Comment:
Let's make the version number 1.23.0. We only do patch releases if we have only bugfixes. This release has many changes including a bunch of features, so a minor version bump makes sense.